src/otx/algo/instance_segmentation/heads/rpn_head.py

```diff
@@ -184,42 +256,13 @@ def predict(
 
         cls_scores, bbox_preds = self(x)
 
-        return self.predict_by_feat(cls_scores, bbox_preds, batch_img_metas=batch_img_metas, rescale=rescale)
-
-    def loss_by_feat(
-        self,
-        cls_scores: list[Tensor],
-        bbox_preds: list[Tensor],
-        batch_gt_instances: list[InstanceData],
-        batch_img_metas: list[dict],
-        batch_gt_instances_ignore: list[InstanceData] | None = None,
-    ) -> dict:
-        """Calculate the loss based on the features extracted by the detection head.
-
-        Args:
-            cls_scores (list[Tensor]): Box scores for each scale level,
-                has shape (N, num_anchors * num_classes, H, W).
-            bbox_preds (list[Tensor]): Box energies / deltas for each scale
-                level with shape (N, num_anchors * 4, H, W).
-            batch_gt_instances (list[InstanceData]): Batch of gt_instance.
-                It usually includes ``bboxes`` and ``labels`` attributes.
-            batch_img_metas (list[dict]): Meta information of each image, e.g.,
-                image size, scaling factor, etc.
-            batch_gt_instances_ignore (list[InstanceData], Optional):
-                Batch of gt_instances_ignore. It includes ``bboxes`` attribute
-                data that is ignored during training and testing.
-
-        Returns:
-            dict[str, Tensor]: A dictionary of loss components.
-        """
-        losses = super().loss_by_feat(
+        return self.predict_by_feat(
             cls_scores,
             bbox_preds,
-            batch_gt_instances,
-            batch_img_metas,
-            batch_gt_instances_ignore=batch_gt_instances_ignore,
+            batch_img_metas=batch_img_metas,
+            rescale=rescale,
+            cfg=self.test_cfg,
```

Comment:
It's just a question. Why wasn't it passed before?

Reply:
It was later implicitly used in another method. So, basically nothing changed. Never mind, I will eliminate this behavior with the following PR.
![image](https://github.com/user-attachments/assets/14373ae4-5644-4757-b6a0-81e36eeb9023)